README.md

```diff
@@ -221,16 +221,86 @@ root
 </pre>
 
 
-
-## Usage
-Todo
-
-### Compiling
-
-- Install go version > 1.8
-
-### Compiling
-
+## Getting Started
+The following section describes building the reference implementation of the Obscuro protocol, running the unit and 
+integration tests, and deploying a local testnet for end-to-end testing. The reference implementation of Obscuro is 
+written in [go](https://go.dev) with the IDE of choice being [GoLand](https://www.jetbrains.com/go/) from Jetbrains. 
+Unless otherwise stated, all paths stated herein are relative to the root of the `go-obscuro` checkout.
+
+
+### Dependencies
+The following dependencies are required to be installed locally;
+
+- [go](https://go.dev) (version > 1.8)
+- [docker](https://docs.docker.com/get-docker/) (recommend latest version)
+
+
+### Building
+To create the build artifacts local to the checkout of the repository the easiest approach is to build each component 
+separately for the host, enclave, and wallet extension  i.e. 
```

Comment:
I'm wondering if it would make sense to create a makefile, or whatever is used in go. 
@otherview, is there a standard thing?

Reply:
I think there should be a flag of `go build` that outputs all the binaries.

But Makefiles are also pretty common yeah.

Reply:
ok, let's raise a ticket to create a makefile to make it easy

Reply:
I have created https://github.com/obscuronet/obscuro-internal/issues/642 to cover this.